packages/ssh-tunnel/src/index.spec.ts

```diff
@@ -106,6 +112,14 @@ async function stopTestSshTunnel() {
   }
 }
 
+async function breakSshTunnelConnection() {
+  try {
+    return once(sshTunnel.sshClient, 'close');
```

Comment:
is this supposed to be 
```suggestion
    return await once(sshTunnel.sshClient, 'close');
```

? Otherwise I’m not sure why there’s a try/finally :slightly_smiling_face:

Reply:
Aah good point.

Reply:
Should be

```javascript
function breakSshTunnelConnection() {
  const promise = once(sshTunnel.sshClient, 'close');
  sshTunnel.sshClient.end();
  return promise;
}
```

No idea where I copied that from :)

Reply:
Aah. Just found it. Will change _that_ as well..